### PR TITLE
Update for Chromium 117.0.5863.0

### PR DESCRIPTION
**Pull request by Raytonne**
Please check [oqs-demos PR #210](https://github.com/open-quantum-safe/oqs-demos/pull/210)

### CODE REVIEW

File: include/openssl/evp.h [hunk 1]
```diff
@@ -234,6 +234,10 @@ OPENSSL_EXPORT EC_KEY *EVP_PKEY_get1_EC_KEY(const EVP_PKEY *pkey);
    (pkey_id == NID_hqc192) || \
    (pkey_id == NID_hqc256) || \
 0 )
+
+OPENSSL_EXPORT int oqs_verify_sig(EVP_PKEY *bssl_oqs_pkey, const uint8_t *sig,
```
Comment (baentsch) [inline include/openssl/evp.h:238]: This is not a good place for the function prototype: The next time `generate.py` is run, this code is deleted (everything between "OQS_TEMPLATE..._START" and "...END" -> Please move this code right before the "// EVP_PKEY_set_type sets..." comment.

**Review by baentsch — CHANGES_REQUESTED** [review]
Thanks for adding the verify function. Please change the place of the function prototype to avoid problems with the OQS integration code generator. Out of curiosity: How/why does this work with hybrid algorithms? The code seems to only cater for pure OQS algorithms...

**Comment by baentsch** [comment]
> All _applicable_ interop tests (with the interop server at test.openquantumsafe.org) have passed.

Can you be specific in which those are? Surely not the x... hybrid KEMs, but what else doesn't work?

**Review by baentsch — APPROVED** [review]

**Comment by dstebila** [comment]
@Raytonne Are you interested in doing another update of this boringssl fork to match the recent algorithm updates in liboqs?

**Comment by pi-314159** [comment]
@dstebila @baentsch  Just tested: current boringssl works with liboqs 0.9.0. Of course, there is no support for McEliece.
Since the interop server at test.openquantumsafe.org doesn't support McEliece, I guess there is no need to add McEliece to boringssl now?

**Comment by baentsch** [comment]
> Just tested: current boringssl works with liboqs 0.9.0. 

Thanks for this check! 

>  there is no need to add McEliece to boringssl now?

Correct. Rationale: https://github.com/open-quantum-safe/oqs-provider/issues/191

**Pull request merged**